### PR TITLE
make triagebot cuter

### PR DESCRIPTION
**Pull request by jyn514**
hi i am new rust user can one of you experienced rust people review my code please

### CODE REVIEW

File: src/handlers/assign.rs [hunk 1]
```diff
@@ -56,17 +56,17 @@ const WELCOME_WITHOUT_REVIEWER: &str = "@Mark-Simulacrum (NB. this repo may be m
 
 const RETURNING_USER_WELCOME_MESSAGE: &str = "r? @{assignee}
 
-({bot} has picked a reviewer for you, use r? to override)";
+({bot} has picked a reviewer for you, use r? to override) (˶˃ᆺ˂˶)";
```
Comment (Nilstrieb) [inline src/handlers/assign.rs:59]: I'm not sure whether this is appropriate here. Having a reviewer is something very positive, and they don't feel that happy to me. Maybe replace it with something more happy? The other ones fit very well though

Comment (jyn514) [reply src/handlers/assign.rs:59]: oh ok is this happy enough like this:

(*・ω・)ﾉ

Comment (Nilstrieb) [reply src/handlers/assign.rs]: that sounds great ferriscat

**Comment by apiraino** [comment]
tbh I find this addition not really necessary. Why?
- Point 1: I (among others) try to recognize the work of contributors on both ends of a PR (author and reviewers) with an explicit "thank you" message, imo more direct and personal than a hardcoded emoji
- Point 2: I don't even know what that means and like maybe others don't

**Pull request closed without merging**